src/components/Books.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 import AddBook from './Addbook';
 
-function Books({ books }) {
+function Books() {
+  const books = useSelector((state) => state.booksReducer);
+
   return (
     <>
       {books.map((book) => (
```

Comment:
- [ ] When I click on `add book`, an error shows up on the console. Please make sure that the books list is fetched from the redux store and rendered on the Books layout.

<img width="698" alt="image" src="https://user-images.githubusercontent.com/77190203/161882038-a26116f6-8f53-4f87-a776-34469fc35238.png">